build.gradle

```diff
@@ -158,21 +135,25 @@ dependencyCheck {
 
 ext {
   log4JVersion = "2.17.0"
+  feignJackson = '11.6'
+  groovyVersion = '3.0.11!!'
+  hamcrestVersion = '1.3'
+  hibernateValidatorVersion = '6.0.20.Final'
+  jacksonVersion = '2.13.2'
+  javaxValidationVersion = '2.0.1.Final'
+  junitJupiterVersion = '5.8.1'
+  junitVintageVersion = '5.7.0'
+  log4jVersion = '2.17.1'
```

Comment:
Appears to be a duplicate - log4jVersion already declared on line 137.

Reply:
removed duplicate

Reply:
Confirmed duplicate has been removed.